selene-vscode/src/extension.ts

```diff
@@ -161,11 +166,11 @@ export async function activate(context: vscode.ExtensionContext) {
     }
 
     function listenToChange() {
-		switch (vscode.workspace.getConfiguration("selene").get<string>("run")) {
-            case "onSave": {
+		switch (vscode.workspace.getConfiguration("selene").get<RunType>("run")) {
+            case RunType.OnSave: {
```

Comment:
You don't need these braces, make sure you run prettier.

Reply:
There is no configuration file for prettier, it ends up formatting each line differently.

Reply:
shame